R/general.R

```diff
@@ -53,6 +53,119 @@ Read10X_vdj <- function(object, data.dir, assay = NULL, force = F, sort.by = c("
     return(ReadData(object, assay = assay, data = data, fields = fields, columns = columns, force = force, sort.by = sort.by))
 }
 
+#' Load 10x VDJ data from multiple directories into a Seurat object containing multiple samples
+#'
+#' @param object Seurat object
+#' @param column (Optional:) Metadata column that contains the information to distinguish samples.
+#' @param data.dir directory containing Cellranger output directories for every sample. If column is specified this will be a named list
+#' @param assay VDJ assay type for loaded data. This is automatically detected from input, but can be overwritten when something goes wrong.
+#' @param force Add VDJ data without checking overlap in cell-barcodes. Default = FALSE
+#' @param sort.by Column to sort the data to determine if chain is primary or secondary. Options = umis, reads
+#' @param use.filtered Load filtered contig annotation. Default = TRUE
+#' @param quiet Ignore warnings. Default = FALSE
+#'
+#' @importFrom dplyr %>% filter left_join
+#' @importFrom rlang .data
+#' @importFrom utils read.csv
+#'
+#' @export
+
+Read10X_MultiVDJ <- function(object, data.dir, column = NULL , assay = NULL, force = F, sort.by = c("umis","reads"), use.filtered = T, quiet = F) {
```

Comment:
Maybe rename `column` to `identity_column`, `sample_column`... or something that is more informative than just `column`. It should be clear from just the function arguments what it doesn, without having to read the documentation